thirdparty/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package thirdparty defines interactions with third-party services.
```

Comment:
Can we mention that the cloud provider is not part of this?

As far as I've seen, this package is really vaguely defined and just contains a random mish-mash of unrelated glue code for stuff that's not written by us.